src/config/index.ts

```diff
@@ -71,3 +73,5 @@ export const CANNOT_SUBMIT_ANOTHER_APPLICATION = `${BASE_COMMON_URL}/cannot-subm
 export const WHAT_IS_YOUR_EMAIL = `${BASE_COMMON_URL}/what-is-your-email/what-is-your-email`;
 export const CANNOT_REGISTER_AGAIN = `${BASE_COMMON_URL}/cannot-register-again/cannot-register-again`;
 export const PAYMENT_FAILED = `${BASE_COMMON_URL}/payment-failed/payment-failed`;
+
+export const UPDATE_ACSP_DETAILS_HOME = `${BASE_UPDATE_ACSP_DETAILS_URL}/index/home`;
```

Comment:
Create one section for "update acsp" and keep new urls together

Reply:
Grouped update acsp details URLs together within index.ts file